tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/BlockingMiddleware.cs

```diff
@@ -81,25 +85,33 @@ internal async Task Invoke(HttpContext context)
     {
         var security = Security.Instance;
         var endedResponse = false;
+
         if (security.Settings.Enabled)
         {
-            var securityCoordinator = new SecurityCoordinator(security, context, (Span)Tracer.Instance.ActiveScope.Span);
-            if (_endPipeline && !context.Response.HasStarted)
-            {
-                context.Response.StatusCode = 404;
-            }
-
-            var result = securityCoordinator.Scan();
-            if (result?.ShouldBeReported is true)
+            if (Tracer.Instance?.ActiveScope?.Span is Span span)
```

Comment:
I couldn't help thinking about Monty Python here 😂

Reply:
span, span, double egg chips and span ;)